docs/modules/recipes/pages/orm/prisma/index.adoc

```diff
@@ -48,154 +59,240 @@ generator client {
+Now we need to create our routes which we will authorize. For this simple example, we will create a `GET` for a contact resource. 
 
-Using the Prisma client, query for the contact which matches the ID of the URL parameter. If it is not found, return an error message.
+Using the Prisma client, query for the contact which matches the ID of the URL parameter. If it is not found, return an error message. Add the following to `src/index.ts`:
 
 [source,js]
 ----
-app.get("/contacts/:id", async (req, res) => {
+// Implementing an authentication provider is out of scope of this article and you will more than likely already have one in place,
+// So we build a static one here for indicative use
+const user = {
+  "id": "1", // user id
+  "role": "user", // single role (user, admin)
+  //"role": "admin",
+  "department": "Sales" // department of the user
+  //"department": "Marketing"
+};
+
+app.get("/contacts/:id", async ({ params }, res) => {
   // load the contact
   const contact = await prisma.contact.findUnique({
     where: {
-      id: parseInt(req.params.id),
-    }
+      id: params.id,
+    },
   });
   if (!contact) return res.status(404).json({ error: "Contact not found" });
 
   // TODO check authz and return a response
-
 });
 ----
 
-Implementing an authentication provider is out of scope of this article and you will more than likely already have one in place, so this code is assuming that the req.user object is structured as follows containing information about the user. These fields will be used by Cerbos to authorize actions.
+=== Setting up Cerbos
 
-[source,js]
-----
-{
-  "id": 1 // user id,
-  "roles": ["user"], // list of roles (user, admin)
-  "department": "Sales" // department of the user
-}
-----
-
-== Setting up Cerbos
-As Cerbos is self-hosted the first step is to run an instance locally for development. For this we need to create a few files and a folder to hold the policies.
+As Cerbos is self-hosted, the first step is to run an instance locally for development. For this we need to create a few files and a folder to hold the policies.
 
-Note: We will be using a docker container to run the instance so ensure you have this setup first.
+NOTE: We will be using a docker container to run the instance so ensure you have this setup first.
 
-First create a config folder (https://github.com/cerbos/express-prisma-cerbos/tree/main/cerbos[see repo]) and a file called `config.yaml`. This tells the Cerbos instance which port to run on and where the policies are located:
+First, create a `cerbos` directory (https://github.com/cerbos/express-prisma-cerbos/tree/main/cerbos[see repo]) with the subdirectories; `config` and `policies`. Then create a file `config.yaml` inside the `cerbos` subdirectory. The contents of the config will tell the Cerbos instance which port to run on and where the policies are located. Run the following to create the files, and populate the config:
```

Comment:
With the Docker container, it's not necessary to create the config file. Simply mount the policies under `/policies`.